Sprout/View Controllers/LoginViewController.m

```diff
@@ -57,7 +57,10 @@ - (IBAction)registerUser:(id)sender {
         
         newUser.username = self.usernameField.text;
         newUser.password = self.passwordField.text;
-        
+        newUser[@"completedTasks"] = @0;
```

Comment:
Why do these need to be set? Can't they just be read if they're there and set if they're not? Default for not being there is basically 0 anyway.